invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -252,6 +252,21 @@ <h2 id="files-heading">{{ _('Files') }}</h2>
                               <p>{{ _("Reason") }}: {{ record.access.embargo.reason }}</p>
                             {% endif %}
 
+                            {%- if not current_user.is_anonymous %}
+                            {# TODO allow a similar workflow for guests #}
+                            <hr style="border-color: darkred;">
+                            <p>
+                              <p>
+                                <strong>Request access</strong>
+                              </p>
+                              <p>
+                                Here we would like to display the customized text from the record parent's access settings, if there is one.
+                                Otherwise, we just display a boring old default text.
+                              </p>
+                              <button id="request-access-button" data-recid="{{ record.id }}">Request access</button>
```

Comment:
```suggestion
                              <a class="ui button" id="request-access-button" data-recid="{{ record.id }}">Request access</a>
```
the semantic ui class, for accessibility (screen readers) it is better to use the <a> tag